test/cloudtest/test_upgrade.py

```diff
@@ -45,7 +45,7 @@ def test_upgrade(aws_region: Optional[str]) -> None:
     """Test upgrade from the last released verison to the current source by running all the Platform Checks"""
 
     mz = MaterializeApplication(tag=LAST_RELEASED_VERSION, aws_region=aws_region)
-    wait(condition="condition=Ready", resource="pod/cluster-u1-replica-1-0")
+    wait(condition="condition=Ready", resource="pod/cluster-u1-replica-u1-0")
```

Comment:
The cloudtest upgrade is now failing -- https://buildkite.com/materialize/nightlies/builds/2402#01882e60-c710-42cd-bb33-9a3c127eddf8 . Apparently the pod name is not changed on upgrade, so possibly the old name still applies?